src/app/pages/search/banned-developers/banned-developers-view.jsx

```diff
@@ -16,7 +16,8 @@ import {
 } from 'api/search';
 import {
   ChplLink,
-  ChplPagination,
+  ChplPagination, 
```

Comment:
```suggestion
  ChplPagination,
```

I see spacing! (ref: "I see dead people"... Ask me if that reference is too old 😉)